lib/foods.scss

```diff
@@ -1,5 +1,58 @@
-.foods-table {
-  font-size: 14px;
+%format-table {
```

Comment:
I've never seen this placeholder class used, but I like it!

Does this only go into effect when it's extended (like in lines 15/16)?

Reply:
Yes, it's great for refactoring.